include/nccl_ofi_mr.h

```diff
@@ -9,7 +9,94 @@
 extern "C" {
 #endif
 
+#include "config.h"
 #include <pthread.h>
+#include <assert.h> // for assert
```

Comment:
we don't do the commenting anywhere else, and this always is impossible to maintain.  Please don't.